src/App.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
-import Container from "./containerForm/Container";
+import FormContainer from "./formContainer/FormContainer";
```

Comment:
Please rename component directory so it's consistent with component name (Capitalise first key)

Reply:
Corrected

Reply:
After folder renaming it should look like
import FormContainer from "./**FormContainer**/FormContainer";

Reply:
It looks exactly like that
```suggestion
import FormContainer from "./FormContainer/FormContainer";
```